src/cnxml/de.ts

```diff
@@ -228,6 +228,9 @@ export function children(
     for (const child of el.childNodes) {
         switch (child.nodeType) {
         case Node.ELEMENT_NODE:
+            if (child.nodeType === Node.ELEMENT_NODE && child.childNodes.length === 0) {
+                (child as Element).append('')
+            }
```

Comment:
1. You are already in `case Node.ELEMENT_NODE`, so a check for `child.nodeType === Node.ELEMENT_NODE` is unnecessary;

2. Mutating DOM tree will likely be more expensive than mutating Slate document, so I'd prefer to avoid it. I suggest instead doing

   ```typescript
   if (el.childNodes.length === 0) {
       editor.apply({ type: 'insert_node', path: path.current!, node: { text: '' } })
   }
   ```

   after this loop